.github/workflows/push_pr.yml

```diff
@@ -78,13 +78,18 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
```

Comment:
There is GO_VERSION env variable, can you also update it at line 60 and elsewhere?

By the way I asked in many PRs, so sorry if you read it already, why do we need to setup go now?

Reply:
Sure, thanks!

Reply:
Including this comment to keep in handy: Needed to use golangci-lint-action v3. Review [this](https://github.com/golangci/golangci-lint-action/pull/403) for details.